build.mjs

```diff
@@ -2,17 +2,22 @@ import path from 'path';
 import { fileURLToPath } from 'url';
 import fs from 'fs/promises';
 import process from 'process';
-import { build } from 'vite';
+import { build, loadConfigFromFile } from 'vite';
 import { minify } from 'terser';
 import humanFormat from 'human-format';
 import brotliSize from 'brotli-size';
-import config from './vite.config.mjs';
+
+const config = await loadConfigFromFile({ mode: 'production' });
```

Comment:
Why this change?

Reply:
I moved the vite config to a typescript file, but that means you can't import it as plain js anymore